File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& num, int target) {
        int p,i,j;
        vector<int> ans;
        for(i =0;i<num.size();i++){
            p=target-num[i];
            ans.push_back(i);
            for(j=i+1;j<num.size();j++){
                if(p==num[j])
                {
                   ans.push_back(j); 
                   return ans;
                }
            }
            ans.pop_back();
        }
        return ans;
    }
};

File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ar = new int[2];
        HashMap <Integer, Integer> hm = new HashMap<>();
        
        for(int i=0; i<nums.length; i++){
            if(hm.containsKey(target - nums[i])){
                ar[1] = i;
                ar[0] = hm.get(target - nums[i]);
            }
            hm.put(nums[i], i);        
        }
        return ar;
    }
}